src/java/org/apache/cassandra/config/Config.java

```diff
@@ -81,6 +81,7 @@
     public boolean auto_bootstrap = true;
     public volatile boolean hinted_handoff_enabled = true;
     public Set<String> hinted_handoff_disabled_datacenters = Sets.newConcurrentHashSet();
+    public volatile int max_hint_window_in_ms = 3 * 3600 * 1000; // three hours
```

Comment:
do we need this field here? Wouldn't the annotation in the next like cover this property?

Reply:
No. I think it was added during rebase accidentally. Good catch!